core/src/main/java/jenkins/diagnosis/HsErrPidList.java

```diff
@@ -123,7 +124,7 @@ private void scan(String pattern) {
     private void scanFile(File log) {
         LOGGER.fine("Scanning "+log);
 
-        try (Reader rawReader = new FileReader(log);
+        try (Reader rawReader = Files.newBufferedReader(log.toPath(), Charset.defaultCharset());
```

Comment:
I couldn't find any documentation on the encoding of `hs_err_pid`, but it seems reasonable to preserve existing behavior here.

Reply:
source code [looks like platform default](https://github.com/openjdk/jdk/blob/739769c8fc4b496f08a92225a12d07414537b6c0/test/hotspot/jtreg/runtime/ErrorHandling/TestCrashOnOutOfMemoryError.java#L108-L121) or the test is wrong :)

Reply:
Nice find!